backend/test/mocks.py

```diff
@@ -39,8 +35,23 @@ def create_institution():
     inst_hash = getHash(institution)
     institution.name = "Inst %s" % inst_hash
     institution.email = "inst%s@email.com" % inst_hash
-    institution.members = []
-    institution.followers = []
-    institution.posts = []
-    institution.acronym = inst_hash
+    institution.acronym = "acronym %s" % inst_hash
+    institution.cnpj = "cnpj %s" % inst_hash
```

Comment:
Only create required properties. Leave the user to extend the object once it's necessary.